packages/web-components/src/components/va-checkbox-group/va-checkbox-group.tsx

```diff
@@ -185,6 +191,7 @@ export class VaCheckboxGroup {
             <legend
               class={legendClass}
               aria-describedby={messageAriaDescribedbyId}
+              part="legend"
```

Comment:
The `part` selector was added so that we could style the legend text through the shadow dom bold: 

![Screenshot 2025-04-17 at 4 07 12 PM](https://github.com/user-attachments/assets/510d40d1-54e0-4104-8d7f-62a8e88a122b)